scipy/stats/_continuous_distns.py

```diff
@@ -5109,14 +5109,14 @@ def _fitstart(self, data):
         nu_beta = (p95 + p05 - 2*p50)/(p95 - p05)
 
         if nu_alpha >= 2.439:
-            alpha = np.clip(psi_1(nu_beta, nu_alpha)[0], np.finfo(float).eps, 2.)
+            alpha = np.clip(psi_1(abs(nu_beta), nu_alpha)[0], np.finfo(float).eps, 2.)
```

Comment:
For consistency, I'd suggest handling this in the same way that the current code does by adding a `psi_1_1` function:
https://github.com/scipy/scipy/blob/affdc4d54142cdcfedcfa06e1185c2ed0441c36c/scipy/stats/_continuous_distns.py#L5092-L5099

McCulloch's paper has the negative `nu_beta` case, but it was somehow missed in scipy's implementation.
![image](https://user-images.githubusercontent.com/14023456/127035848-a775baf3-2b14-48d2-8dc1-ded584d5a6c0.png)

The current code essentially treats negative `nu_beta` as `nu_beta == 0.0` in the interpolation, which is incorrect.

Reply:
"I'd suggest handling this in the same way that the current code does by adding a psi_1_1 function"

In terms of consistency, you're probably right.

Any appetite for a slightly more ambitious revision with new helper functions `_rectified()` and `_reflected()` that would result in something like this:

```
def _rectified(func):
    return lambda xval, yval: func(val if xval > 0 else -xval, yval)

def _reflected(func):
    rectified_func = _rectified(func)
    def lookup(xval, yval):
        rectified_result = rectified_func(val,  yval)
        return rectified_result if xval > 0 else -rectified_result
    return lookup

psi_1_1 = _rectified(interpolate.interp2d(nu_beta_range, nu_alpha_range, alpha_table))
psi_2_1 = _reflected(interpolate.interp2d(nu_beta_range, nu_alpha_range, beta_table))

phi_3_1 = _rectified(interpolate.interp2d(beta_range, alpha_range, nu_c_table))
phi_5_1 = _reflected(interpolate.interp2d(beta_range, alpha_range, nu_zeta_table))

```

Or is it better to go with the minimal fix--less risk of breaking something that currently works?

Reply:
Thanks for the excerpts from McCulloch--I haven't tracked down a copy yet.  McCulloch's equation 4.6 for the delta estimator suggests that the adjustment to zeta should be applied when alpha is **not** 1.  This is exactly the opposite of what the code is doing (I show it here with spurious clipping removed):

`delta = zeta-beta*c*np.tan(np.pi*alpha/2.) if alpha == 1. else zeta`

So I think the `==` should be `!=`.  Is that right?

If the true value could lie anywhere when alpha is exactly 1, there is arguably no point in checking for this event which is vanishingly unlikely anyway.  So maybe the code should just be:

`delta = zeta-beta*c*np.tan(np.pi*alpha/2.)`

Thoughts?

Reply:
I think you're right, but I'll have to check more closely tonight. There are many different parameterizations of stable laws and the literature has historically been very inconsistent in this regard.

I know scipy's current code uses Nolan's S1 parameterization and much of the theoretical work in papers used to use his S0 parameterization, which requires a shift of exactly the form used in the delta estimate here. It's unlikely, but that *could* explain the difference in the code.

For instance, Nolan notes the S0 <-> S1 transformation requires the following shift
![image](https://user-images.githubusercontent.com/14023456/127168112-758500a3-d06f-474f-ad3d-b788817c47e9.png)
(Also note that the field never really converged on consistent notation. Above, gamma plays a role similar to McCulloch's "c" in your code).

Reply:
It looks like swapping `alpha == 1` to `alpha != 1` is all that's needed for the delta estimation issue. More details are in https://github.com/scipy/scipy/pull/14476#issuecomment-887853522.